File: Medii de proiectare si programare/java/lab3/src/main/java/com/example/temampp/repositoryes/interfaces/RepoEmployee.java
package com.example.temampp.repositoryes.interfaces;

import com.example.temampp.domains.EmployeeAtOffice;

public interface RepoEmployee {
    boolean logIn(String email, String password);

    void logOut();

    EmployeeAtOffice findOneByEmailAndPassword(String email, String password);
}

File: Medii de proiectare si programare/C#/lab6/temaCSharp/repositoryes/interfaces/RepoEmployee.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using temaCSharp.domains;

namespace temaCSharp.repositoryes
{
    internal interface RepoEmployee: IRepository<int, EmplyeeAtOffice>
    {
        bool login(String email, String password);

        EmplyeeAtOffice logout();
        EmplyeeAtOffice findOneByEmailAndPassword(String email, String password);
    }
}
